hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -363,6 +379,9 @@ def rtc_sentinel_gamma(safe_dir: str, resolution: float = 30.0, radiometry: str
         output_tif = f'{product_name}/{product_name}_{pol.upper()}.tif'
         if scale == 'power':
             shutil.copy(power_tif, output_tif)
+        elif scale == 'decibel':
+            decibel_tif = create_decibel_tiff(power_tif)
```

Comment:
Only one `f` in the function name
```suggestion
            decibel_tif = create_decibel_tif(power_tif)
```